Gopkg.lock

```diff
@@ -381,10 +381,9 @@
   revision = "442624c9ec9243441e83b374a9e22ac549b5c51d"
 
 [[projects]]
-  branch = "master"
   name = "golang.org/x/sys"
   packages = ["unix","windows","windows/registry"]
-  revision = "53aa286056ef226755cd898109dbcdaba8ac0b81"
```

Comment:
@praveenkumar The reason the sha changed because `golang.org/x/sys/` was never used before your PR :)
`pkg/minikube/cluster/cluster_windows.go:	"golang.org/x/sys/windows/registry"`

Reply:
@LalatenduMohanty That piece I know, but if you even try to remove that from `pkg/minikube/cluster/cluster_windows.go` and again do make clean and make it will change the `input-digest`, I did try that also.

Reply:
@praveenkumar I did that experiment, in #1822 as it also uses `sys` and it did not generate the inputs-digest.